tests/unit/drivers/test_matches2doc_rank_drivers.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+
+from jina.drivers.rank import Matches2DocRankDriver
+from jina.executors.rankers import Chunk2DocRanker
+
+from jina.proto import jina_pb2
+
+
+class MockAbsoluteLengthRanker(Chunk2DocRanker):
```

Comment:
Do we want it to use Chunk2DocRanker or simple BaseRanker?

Reply:
Apparently, we want to use `Match2DocRanker`

Reply:
yes, but interface will be exactly as BaseRanker right? u can add an extra inheritance layer if you want, but I do believe that Chunk2DocRanker (although it could somehow work) is missleading.

Reply:
Yes, `BaseRanker` is enough

Reply:
But the interface tells you what exactly are the to-expected arguments. When I want to implement a Ranker in the hub, that works well with the `Matches2DocRankDriver`, it makes the life much easier, if there is an explicit interface, that I have to implement, instead of guessing the right function arguments.

Reply:
@maximilianwerk do u mean is better to add `Match2DocRanker`? For me it is even better than `BaseRanker`, I am just saying that `Chunk2DocRanker` would be missleading.

Reply:
And that's what you put now so good for me

Reply:
I fixed it to `Match2DocRanker` and made a proper documentation of how to use it.

Reply:
please :+1: once again